README.md

```diff
@@ -24,52 +22,72 @@ To get a development environment running on your local machine:
 cp .env.example .env
 ```
 
-3. Install packages
+### Install packages
 
 ```bash
 npm install
 ```
 
-4. Database Setup
-   You'll need to create a free database with [Xata](https://xata.io/) which runs on Postgres. When you create your db, make sure enable the checkbox that says `Enable direct access to Postgres`.
+### Database Setup
+
+To get started, you'll need to create a **free** account with [Xata](https://xata.io/).
+
+Xata runs on Postgres, but to get access to the Postgres layer, youll need to enable this in your workspace settings. Find your workspace settings and toggle to enable `Direct Access to Postgres (BETA)`.
+
+Now, you'll need to create a new database. Give it a name, and make sure to enable the checkbox that says `Enable direct access to Postgres`.
+
+> In this project, we'll be connecting to our db in two different ways: using Prisma ORM and the Xata client SDK. All standard CRUD (Create, read, update, and delete) db interactions will be run through Prisma. We'll only be using the Xata client to take advantage of Xata specific features like search and file uploads.
```

Comment:
This still isn't clear to me what option we should select in the dropdown. Make it explicit. Based off of this I wouldn't know to select the Xata SDK or Prisma ORM (SQL).

Reply:
What dropdown options are you referring to? I'm thinking this might be part of the dashboard onboarding experience that I'm not going through personally. Can you share a screenshot?

Reply:
If I have no other DBs, it takes me directly to the workflow to create one. At one point, it asks how I'd like to connect.

![Screenshot of Xata database creation screen](https://github.com/Learn-Build-Teach/deals-for-devs/assets/40150036/85a9aa6c-7718-4a54-b29a-9168445449ec)